server/secret.go

```diff
@@ -1,32 +1,46 @@
 package server
 
 import (
+	"net/http"
+
 	"github.com/drone/drone/model"
 	"github.com/drone/drone/router/middleware/session"
 	"github.com/drone/drone/store"
 
 	"github.com/gin-gonic/gin"
 )
 
+func GetSecrets(c *gin.Context) {
+	repo := session.Repo(c)
+	secrets, err := store.GetSecretList(c, repo)
+
```

Comment:
yes let's include ID just in case. I might use it in the UI

maybe we do something like this:

```
func (s *Secret) Clone() *Secret {
	return &Secret {
		ID: s.ID,
		Name: s.Name,
		Images: s.Images,
		Events: s.Events,
	}
}
```

and then in the handler we can do something like this:


```
var results []*Secret
for _, s := range secrets {
   results = append(results, s.Clone())
}
```